src/hooks/usePodOwnerProfile/index.js

```diff
@@ -25,14 +25,16 @@ import { joinPath } from "../../stringHelpers";
 import useAuthenticatedProfile from "../useAuthenticatedProfile";
 import usePodRootUri from "../usePodRootUri";
 import useFetchProfile from "../useFetchProfile";
+import usePodOwner from "../usePodOwner";
 
 export default function usePodOwnerProfile() {
   const [profile, setProfile] = useState();
   const [error, setError] = useState();
   const router = useRouter();
-  // const decodedResourceUri = decodeURIComponent(router.query.iri);
   const podRoot = usePodRootUri(router.query.iri, null);
-  const profileIri = podRoot && joinPath(podRoot, "profile/card#me"); // we won't need to do this once ownership is available
+  const { podOwnerWebId } = usePodOwner({ resourceIri: router.query.iri }); // passing in an object for testing purposes
+  const profileIri =
+    podOwnerWebId || (podRoot && joinPath(podRoot, "profile/card#me")); // we won't need to do this once ownership is available
```

Comment:
With this logic, we end up trying to load `podRoot/profile/card#me` first. If you exposed a `requestInProgress` variable from `usePodOwner`, you would be able to default this to null and wait for the request to complete before loading the fallback.